site/cn/index.html

```diff
@@ -111,6 +111,7 @@
                             <li><a href="../ja/index.html">日本語</a></li>
                             <li><a href="../pt-br/index.html">Português-Brasil</a></li>
                             <li><a href="../ru/index.html">Русский</a></li>
+							<li><a href="../hi/index.html">हिंदी</a></li>
```

Comment:
fix the indentation. same comment for files of other language

Reply:
fixed